### PR TITLE
[ML] Include model definition install status for Pytorch models

### PR DESCRIPTION
**Pull request by davidkyle**
Add a new include flag `definition_status` to the GET trained models API. 

```
GET _ml/trained_models/<MODEL_ID>?include=definition_status
```

When the flag is present the trained model configuration returned in the response will have the new boolean field `fully_defined` if the full model definition is present.

The API is not compatible with wildcards or multiple model Ids. `model_id` must map to a single model with `model_type == PYTORCH`

### CODE REVIEW

File: docs/reference/ml/trained-models/apis/get-trained-models.asciidoc
Comment (droberts195) [inline docs/reference/ml/trained-models/apis/get-trained-models.asciidoc:937]:
```suggestion
This field is only present if `include=download_status` was specified in the request.
```

**Review by droberts195 — APPROVED** [review]
LGTM if you could just fix the incomplete sentence in the docs before merging.

**Comment by davidkyle** [comment]
I wasn't happy with `is_fully_downloaded` as some models are *uploaded*. 

I've changed the request parameter from `download_status` -> `definition_status`
And the body response from `is_fully_downloaded` -> `fully_defined`

**Comment by davidkyle** [comment]
@elasticmachine update branch

**Pull request merged**
